etc/cp-schema-registry/values.yaml

```diff
@@ -19,6 +19,7 @@ resources:
 ingress:
   enabled: true
   annotations:
+    kubernetes.io/ingress.class: nginx
```

Comment:
is this change related to kratos and hydra?

Reply:
This change has been added to the schema registry chart and I don't think it should be here.
https://github.com/RADAR-base/cp-helm-charts/commit/44e01484a557b085994ae4687d9b054b40cdc6e5

Reply:
Ah yes, removed this now.